spigot/src/main/java/com/sekwah/advancedportals/spigot/tags/ConditionsTag.java

```diff
@@ -0,0 +1,147 @@
+package com.sekwah.advancedportals.spigot.tags;
+
+import com.sekwah.advancedportals.core.connector.containers.PlayerContainer;
+import com.sekwah.advancedportals.core.registry.TagTarget;
+import com.sekwah.advancedportals.core.util.InfoLogger;
+import com.sekwah.advancedportals.core.util.Lang;
+import com.sekwah.advancedportals.core.warphandler.ActivationData;
+import com.sekwah.advancedportals.core.warphandler.Tag;
+import com.sekwah.advancedportals.shadowed.inject.Inject;
+import me.clip.placeholderapi.PlaceholderAPI;
+import org.bukkit.Bukkit;
+import org.bukkit.entity.Player;
+
+public class ConditionsTag implements Tag.Activation, Tag.Split, Tag.Creation {
+
+    @Inject
+    private InfoLogger infoLogger;
+
+    @Override
+    public boolean preActivated(TagTarget target, PlayerContainer player, ActivationData activeData, String[] argData) {
+
+        for (String condition : argData) {
+            if (!checkConditions(condition, Bukkit.getPlayer(player.getUUID()))) {
+                player.sendMessage(Lang.getNegativePrefix() + Lang.translate("tag.conditions.fail"));
+                return false;
+            }

```

Comment:
Would be a big help if the offending condition would be specified.

^ Even better if all of them would be specified, but I can already imagine that means a big rewrite of `checkConditions` and/or possibly something else too, so while specifying the 1st offender would still be annoying (like back in Window$ XP when copying/moving/deleting/renaming in Windows Explorer), it's a whole lot better than just `Conditions are invalid`.